### PR TITLE
Adjust to dap_server changes to send errors via DAP vs stderr.

### PR DESCRIPTION
**Pull request by noppej**
See https://github.com/probe-rs/probe-rs/pull/2457

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -108,29 +105,30 @@ function logToConsole(consoleMessage: string, fromDebugger: boolean = false) {
...
+    } else if (consoleMessage.startsWith(ConsoleLogSources.console.toLowerCase())) {
         vscode.debug.activeDebugConsole.appendLine(consoleMessage);
     } else {
         switch (consoleLogLevel) {
             case ConsoleLogSources.debug: //  Log Info, Error AND Debug
                 if (
-                    consoleMessage.includes(ConsoleLogSources.console.toLowerCase()) ||
-                    consoleMessage.includes(ConsoleLogSources.error) ||
-                    consoleMessage.includes(ConsoleLogSources.debug)
+                    consoleMessage.startsWith(ConsoleLogSources.console.toLowerCase()) ||
```
Comment (bugadani) [inline src/extension.ts]: `probe-rs-debug` is already lower case, why is there a `.toLowerCase()` transform on it in multiple (at least 7) places?

Comment (noppej) [reply src/extension.ts]: A hang over from when that string was uppercase. Good spotting ... I will fix it.

Comment (noppej) [reply src/extension.ts]: Done :)

**Review by bugadani — APPROVED** [review]
Thank you! Not sure if you want to wait for the companion PR so I refrain from pushing the Big Green Button for now.

**Pull request merged**